galera/files/debian.cnf_slave

```diff
@@ -0,0 +1,13 @@
+{%- from "galera/map.jinja" import slave with context %}
```

Comment:
Just a small hint: you can strip all whitespaces at file's first line by using `-%}`